docs/cambridge.md

```diff
@@ -5,10 +5,58 @@ To use, run the pipeline with `-profile cambridge`. This will download and launc
 with a setup suitable for the Cambridge HPC cluster. Using this profile, either a docker image containing all of the required software will be downloaded,
 and converted to a Singularity image or a Singularity image downloaded directly before execution of the pipeline.
 
-The latest version of Nextflow is not installed by default on the Cambridge HPC cluster. You will need to install it into a directory you have write access to.
-Follow these instructions from the Nextflow documentation.
+### Install Nextflow
 
-- Install Nextflow : [here](https://www.nextflow.io/docs/latest/getstarted.html#)
+The latest version of Nextflow is not installed by default on the Cambridge HPC cluster CSD3. You will need to install it into a directory you have write access to.
+Follow [these instructions](https://www.nextflow.io/docs/latest/getstarted.html#) from the Nextflow documentation.
+
+```
+# move to desired directory on HPC
+cd /home/<username>/path/to/dir
+# get the newest version
+wget -qO- https://get.nextflow.io | bash
+```
+
+### Obtain updated java version
+
+The java version on the HPC needs updating.
+
+```
+cd ~/
+wget https://download.oracle.com/java/20/latest/jdk-20_linux-x64_bin.tar.gz
+tar xvfz jdk-20_linux-x64_bin.tar.gz
+# add these lines to .bashrc
+export JAVA_HOME=/home/ef479/jdk-20.0.1
+export PATH=/home/ef479/jdk-20.0.1/bin:$PATH
```

Comment:
This also seems dangerous, what if someone needs to use a different tool that isn't v20 compatible? Another reason why I suggest writing Conde instructions above (and install mxf into a separate env)

Reply:
I have updated this.